antiplag/antiplag/tasks.py

```diff
@@ -0,0 +1,19 @@
+from celery import shared_task
+from models import *
+from text_comparison import text_comparison
+import json
+
+@shared_task(name='antiplag.tasks.generate_result')
+def generate_result(document, elastic_result):
+    # One document to check against,
+    #up to 10 documents from elastic
+    corpus = document
```

Comment:
Why do you assign document to corpus?

Reply:
@DeMensa123 The text_comparison function needs a parameter where the corpus contains all documents (the one to be compared along with the documents we're comparing it to), this way we append them into one corpus which will go into the function as a parameter